### PR TITLE
Get single inventory item ticket26

### PR DESCRIPTION
**Pull request by zahrafalak**

### CODE REVIEW

**Comment by ryn-ngn** [comment]
Good job Falak! I just have 1 suggestion: can we have similar naming style for the newly exported function, like getItemById or something similar; or getSingleInventoryItem like this branch name. What do you think?

**Comment by zahrafalak** [comment]
Variable name changed to getItemById

**Review by ryn-ngn — APPROVED** [review]
Thanks Falak!

File: controllers/inventory-controller.js [hunk 1]
```diff
@@ -37,6 +37,39 @@ const getAllInventories = async (_req, res) => {
...
+        "inventories.description",
+        "inventories.category",
+        "inventories.status",
+        "inventories.quantity"
+      )
+      .where("inventories.id", req.params.id);
+
+    if (itemFound.length === 0) {
+      return res.status(404).json({
+        message: `Inventory item with ID ${req.params.id} not found`,
```
Comment (rjtbansal) [inline controllers/inventory-controller.js:59]: Great that we are handling on BE but its not being handled on FE. I tried reloading an inventory with an invalid Id. I did get 404 in the network tab but the UI was in permanent loading state. 
We should handle such API responses gracefully on FE. 
image

**Review by rjtbansal — COMMENTED** [review]
Well done! Just had a comment that needed to be done on FE for good UX

**Pull request merged**